tool/modules/uds_fuzz.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/env python
+# Released under GNU General Public License v3
+# https://github.com/Cr0wTom/UDS-Seed-Randomness-Fuzzer
```

Comment:
This kind of link should not be included in modules

Reply:
Yes, sorry I forgot to remove this specifically